shared/js/background/redirect.es6.js

```diff
@@ -66,10 +66,6 @@ function handleRequest (requestData) {
          * Tracker blocking
          * If request is a tracker, cancel the request
          */
-        if (window.chrome) {
-            // this can send an error message when the popup is not open. check lastError to hide it
-            chrome.runtime.sendMessage({'updateTabData': true}, () => chrome.runtime.lastError)
-        }
 
```

Comment:
This was an old remnant that doesn't actually need to be there. It was causing every single request to rerender the popup, including those from background tabs that aren't trackers.